recipe/build_pytorch.sh

```diff
@@ -9,5 +9,5 @@ if [[ "$megabuild" == true ]]; then
     fi
   done
 else
-  $PREFIX/bin/python -m pip install torch-*.whl
+  $PREFIX/bin/python -m pip install --no-deps torch-*.whl
```

Comment:
one question i have is that I thought that the `--no-deps` flag was added to the conda-build system automatically. what failed?

Reply:
There is a long-standing conda-build bug that it doesn't get added in the case of outputs.

Reply:
i understand now

Reply:
https://github.com/conda/conda-build/issues/3993

Reply:
subscribed thanks.